vpr/src/place/move_utils.cpp

```diff
@@ -28,6 +28,51 @@ void set_placer_breakpoint_reached(bool flag) {
     f_placer_breakpoint_reached = flag;
 }
 
+/**
+ * @brief Adjust the search range based on the block type and constraints
+ * 
```

Comment:
Should explain a bit about why and how you adjust the search range.

Reply:
Added the following:

```
* If the block is an IO block, we expand the search range cover the whole column
 * We found empirically that this is a good strategy for IO blocks given they are located in
 * the periphery for most FPGA architectures
```